frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -295,4 +417,30 @@ function BookInfo:getCurrentPageLineWordCounts()
     return lines_nb, words_nb
 end
 
+function BookInfo:showCustomMenu(file, book_props, metadata_updated_caller_callback)
+    local button_dialog
+    local buttons = {{
+        {
+            text = self.custom_book_cover and _("Reset cover image") or _("Set cover image"),
+            align = "left",
+            font_face = "smallinfofont",
+            font_size = 22,
+            font_bold = false,
```

Comment:
Any reason to not use the default values (that I used in PageBrowser and my DictQuickLookup experiments) ?